src/MoBi.Core/CoreRegister.cs

```diff
@@ -71,7 +71,7 @@ public override void RegisterInContainer(IContainer container)
          container.Register<IProjectRetriever, MoBiProjectRetriever>();
          container.Register<IHistoryManager, MoBiHistoryManager>();
          container.Register<IObjectIdResetter, ObjectIdResetter>();
-         container.Register<ISetParameterTask, QuantityTask>();
+         container.Register<ISetParameterTask, ParameterTask>();
```

Comment:
I have created a dummy task calling the right method. This was getting too confusion in Mobi